packages/sdk/src/libs/clients/erc20/README.md

```diff
@@ -0,0 +1,23 @@
+# UMA SDK ERC20 Client
+
+Client to interface with ERC20 style tokens, based on Ethers and typechain.
+
+## Usage
+
+```js
+import {ethers} from 'ethers'
+import * as uma from '@uma/sdk'
+
+// assume you have a url injected from env
+const provider = new ethers.providers.WebSocketProvider(env.CUSTOM_NODE_URL)
+
+// get the contract instance
+const erc20Address:string = // assume you have an emp address you want to connect to
+const erc20Instance:uma.clients.erc20.Instance = uma.clients.erc20.connect(erc20Address,provider)
+// gets all emp events
+const events = await erc20Instance.queryFilter({})
+
+// returns EventState, defined in the emp client
+const state:uma.clients.erc20.EventState = uma.clients.erc20.getEventState(events)
```

Comment:
can this do more than just return events? should this also be documented?

Reply:
yea i should!

Reply:
added more comments, but the client is pretty light, basicaly returns a contract instance, events and will decode events into some state